flit-client/src/types.tsx

```diff
@@ -0,0 +1,11 @@
+export type Message = {
+    source: Account,
+    content: string,
+    timestamp: number, // epoch time in s
+}
+
+export type Account = {
+    name: string,
+    // avatar: Image - how will we represent this?
+    // add in whatever auth-related info we need
+}
```

Comment:
Generally I'm not a fan of creating catch-all files like this, but I think this is the right thing to do here - I think the more complete version of these will end up being typed by the GraphQL schema, and we don't want to block ourselves waiting for that.